src/platform/silabs/SilabsConfig.h

```diff
@@ -112,6 +112,14 @@ class SilabsConfig
     static constexpr Key kConfigKey_ProductURL            = SilabsConfigKey(kMatterFactory_KeyBase, 0x11);
     static constexpr Key kConfigKey_PartNumber            = SilabsConfigKey(kMatterFactory_KeyBase, 0x12);
     static constexpr Key kConfigKey_UniqueId              = SilabsConfigKey(kMatterFactory_KeyBase, 0x1F);
+    static constexpr Key kConfigKey_Creds_KeyId           = SilabsConfigKey(kMatterFactory_KeyBase, 0x20);
```

Comment:
We have do a migration for this. If any app updates to this version from 1.1 or 1.0 they will loose the means to retrieve their credentials.

should be easy. Add an entry in `migrationTable` `(src/platform/silabs/MigrationManager.cpp)` 

your migration function will just be if old key exist, read it and write to new key, delete old key.

Reply:
Btw @jmartinez-silabs, Can we totally remove/ignore this migration tools if we start a firmware at 1.1 ?

Reply:
The migrations table will go over what needs to be migrated and see that it has nothing to do and then save the value of what "migration group" it has validated so it doesn't have to check on future boots.  

I would keep the functionality in your products for future version

Reply:
It might have been ok to add the SilabsConfig to the dependency list for the migration class. After ,  most migration will be tied to persistent storage so either KVS or Silabs Config so not that many other deps. But what you did is good too. Thanks Ricardo